src/utils/test/load-current-session.test.ts

```diff
@@ -74,7 +75,7 @@ describe('loadCurrentSession', () => {
     const session = new Session(`test-shop.myshopify.io_${jwtPayload.sub}`);
     await expect(Context.SESSION_STORAGE.storeSession(session)).resolves.toEqual(true);
 
-    await expect(loadCurrentSession(req, res)).resolves.toEqual(session);
+    await expect(loadCurrentSession(req, res, true)).resolves.toEqual(session);
```

Comment:
Is the `true` necessary, since it's the default?

Reply:
Not particularly!